jans-pycloudlib/jans/pycloudlib/wait.py

```
@@ -161,10 +161,19 @@ def wait_for_secret(manager: Manager, **kwargs: _t.Any) -> None:
+    # check following entries (shouldn't be empty) to mark whether secrets are ready
+    entries_ready = all([
+        # required for secure connection
+        manager.secret.get("ssl_cert"),
+        # required for obfuscating plain text
+        manager.secret.get("encoded_salt"),
+        # required for signing keys
+        manager.secret.get("auth_jks_base64"),
+    ])
+
+    if not conn_only and not entries_ready:
+        raise WaitError("Secret 'ssl_cert', 'encoded_salt', and 'auth_jks_base64' are not available")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)